static/app/utils/fields/index.ts

```diff
@@ -789,6 +788,80 @@ export const AGGREGATION_FIELDS: Record<AggregationKey, FieldDefinition> = {
   },
 };
 
+// TODO: Extend the two lists below with more options upon backend support
+export const ALLOWED_EXPLORE_VISUALIZE_FIELDS: SpanIndexedField[] = [
+  SpanIndexedField.SPAN_DURATION,
+  SpanIndexedField.SPAN_SELF_TIME,
+];
+
+export const ALLOWED_EXPLORE_VISUALIZE_AGGREGATES: AggregationKey[] = [
+  AggregationKey.COUNT,
+  AggregationKey.MIN,
+  AggregationKey.MAX,
+  AggregationKey.AVG,
+];
```

Comment:
This is fine for now, but we may run into complications as we add support for numeric span attributes which will be fetched dynamically.